src/Cassandra.Tests/Mapping/Pocos/HairColor.cs

```diff
@@ -2,12 +2,12 @@
 {
     public enum HairColor
     {
-        Black,
-        Blonde,
-        Brown,
-        Gray,
-        Red,
-        White,
-        Other
+        Black = 0,
```

Comment:
Maybe I am overseeing something, but why set the default value?

Reply:
I've added some tests that check the value of the enum based on their int representation. Its just to prevent issues in the future when adding a new member in the enum.